### PR TITLE
Arabic Documentation on Using Crowdin

### PR DESCRIPTION
**Pull request by rnabage**
- #13

### CODE REVIEW

**Comment by rnabage** [comment]
Hello @BatoolMM I've submitted this PR, could you please review and let me know if I should make any corrections? Thank you.

File: docs/ruqayyah_docs.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+## مشاكل واجهتها
+- عندما بدأت في الترجمة، أي وقت أرى مصطلحاً تقنياً لا أعرف كيف أترجمه كان عليّ الذهاب إلى نافظة مختلفة والبحث في ويكيبيديا عنه. حتى نقرت على كلمة لإضافتها إلى المسرد، ولاحظت خيار البحث في المصطلحات و ويكيبيديا. ضغطت عليه وقدم لي صفحة ويكيبيديا في لوحتي الجانبية اليمنى.
+<img width="1440" alt="Screenshot5" src="https://user-images.githubusercontent.com/58336721/195917358-cae7d6e3-57d5-4e4d-bc76-bcc574b3157f.png">
+ أصبحت هذه ميزتي المفضلة في محرر كراودِن لأنها ملائمة جداً وتقلل من وقت عملي.
+- عندما بدأت الترجمة شعرت بالحيرة فيما إذا كان ينبغي عليّ ترجمة حرفية لأن المتكلمين كانوا في بعض الأحيان غير مكتملين جملهم أو كرروا أنفسهم. مشكلة أخرى واجهتها هي أنّ بعض السلاسل المكتوبة للترجمة لم تكن نفس الشىء مع ما يقوله المتكلم. وبعض السلاسل تم قطعها و تقسيمها إلى أجزاء بدلًا من جمل مستمرة. كل هذه المشاكل تم حلها ببساطة عندما سألت المرشدة(mentor) في تعليق عند المشاكل المحددة. قدمتني نصائح قيّمة جدًا بأنني يجب أن أترجم السياق وليس المعنى الحرفي. وأيضًا زادت أنني يجب أن أترجم كلام المتكلم أولاً قبل كل شيء. هذا أعطاني فكرة الإستماع إلى بعض السلاسل معًا قبل وضع ترجمة.
```
Comment (BatoolMM) [inline docs/ruqayyah_docs.md]: The alignment here is LTR, not RTL!

**Review by BatoolMM — CHANGES_REQUESTED** [review]

**Comment by rnabage** [comment]
Hello @BatoolMM I've effected the changes. Thank you!

**Comment by BatoolMM** [comment]
This looks fantasic, thank you @rnabage!

**Comment by BatoolMM** [comment]
@all-contributors please add @rnabage for documentation and translation

**Pull request merged**